plugins/services/src/js/structs/__tests__/PodInstance-test.js

```diff
@@ -52,6 +52,32 @@ describe("PodInstance", function() {
     });
   });
 
+  describe("#getAgentRegion", function() {
+    it("returns the correct value", function() {
```

Comment:
Can we have a more descriptive title for all tests, please? e.g https://github.com/dcos/dcos-ui/pull/3154/files#diff-898ccecc5fbc3897c2529139b461bcccR78